pom.xml

```diff
@@ -5,6 +5,7 @@
 
   <properties>
     <version>2.0.0</version>
+    <minecraft.api.version>1.18</minecraft.api.version>
     <maven-version>${version}-DEV-UNOFFICIAL</maven-version>
```

Comment:
I would prefer using the same naming pattern/convention here. Personally i would use the `-` variant. more common would be `.`.

And then this may would result in `version.maven` and `version.minecraft.api`, or the same with dashes. But make it consistent